src/Mailer.js

```diff
@@ -108,7 +108,7 @@ export default class Mailer extends templater.Templater {
     // @@override
     build(next) {
 
-        if (!this._to) {
+        if (!this._to && !this.cc && !this.bcc) {
```

Comment:
update also the error message, also you should be checking on this._cc and this._bcc as this will only have values upon Template.render

Reply:
resolved in 
2fb7da8